rows.go

```diff
@@ -2,13 +2,22 @@ package athena
 
 import (
 	"database/sql/driver"
+	"fmt"
 	"io"
+	"strings"
+
+	"github.com/sirupsen/logrus"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/athena"
 	"github.com/aws/aws-sdk-go/service/athena/athenaiface"
 )
 
+var listOfFailedQueryResult = []string{
```

Comment:
Taken a route of matching an error string with resultset. This may not be an exhaustive list, we can keep on adding to this list for the resultset which we think is an error.